pkg/iac/terraform/state/backend/http_reader.go

```diff
@@ -31,6 +33,16 @@ func NewHTTPReader(rawURL string, opts *Options) (*HTTPBackend, error) {
 		return nil, err
 	}
 
+	buf := new(bytes.Buffer)
+	_, err = buf.ReadFrom(res.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	if res.StatusCode < 200 || res.StatusCode >= 400 {
+		return nil, fmt.Errorf("error in backend HTTP(s): non-200 OK status code: %s body: \"%s\"", res.Status, buf.String())
```

Comment:
You should use `github.com/pkg/errors` pkg instead of `fmt`. `fmt` is banished from the project since it does not wrap the stack and thus it is very complicated to understand from sentry.

You can use `errors.Errorf()` here